src/Functions/FunctionsJSON.cpp

```diff
@@ -1332,13 +1383,69 @@ class JSONExtractRawImpl
 
     static bool insertResultToColumn(IColumn & dest, const Element & element, std::string_view)
     {
-        ColumnString & col_str = assert_cast<ColumnString &>(dest);
-        auto & chars = col_str.getChars();
-        WriteBufferFromVector<ColumnString::Chars> buf(chars, AppendModeTag());
+        if (dest.getDataType() == TypeIndex::LowCardinality)
```

Comment:
But `JSONExtractRaw` function always returns `String`, is it possible that destination column is `LowCardinality`?